scripts/cmake/Modules/LibAddPlugin.cmake

```
@@ -637,6 +637,8 @@ function (add_plugin PLUGIN_SHORT_NAME)
 
 	if (BUILD_SHARED)
 		add_library (${PLUGIN_NAME} MODULE ${ARG_SOURCES} ${ARG_OBJECT_SOURCES})
+		# switch to -plugin- in CMake targets
```

```suggestion
		# TODO: switch to -plugin- in CMake targets
```

so we can actually find the comment ;)